git-tar/function/ops.go

```diff
@@ -183,7 +244,19 @@ func clone(pushEvent sdk.PushEvent) (string, error) {
 		return "", fmt.Errorf("cannot create user-dir: %s", userDir)
 	}
 
-	git := exec.Command("git", "clone", pushEvent.Repository.CloneURL)
+	at := &authToken{
```

Comment:
Can we call `at` something like `authToken` and `authToken` something else if needed. `githubAuthToken` / `installationAuthToken`?